ddtrace/logger.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+from ddtrace.internal.utils.formats import asbool
+
+
+DEFAULT_FILE_SIZE_BYTES = 15 << 20  # 15 MB
+
+
+def configure_ddtrace_logger():
+    # type: () -> None
+    """ """
+    debug_enabled = asbool(os.environ.get("DD_TRACE_DEBUG", "false"))
+    log_path = os.environ.get("DD_TRACE_LOG_FILE")
+    max_file_bytes = int(os.environ.get("DD_TRACE_FILE_SIZE_BYTES", DEFAULT_FILE_SIZE_BYTES))
+    num_backup = 1
```

Comment:
@wantsui maybe we can add a comment above `max_file_bytes` explaining why we configure `max_file_bytes` and `num_backup = 1` ?

@majorgreys the reason we decided to do this is because if we write to a file we want to cap the max amount of data we can write to disk. if a user forgets that they enabled this and then 50gb of ddtrace debug logs later they run out of disk space, it isn't a great experience for anyone. So we cap the max file size that can happen, and then we _have_ to set a number of backups > 0 (so it can rotate the files when max size is reached), so we decided on 1 just to keep it simple.

Because of that, I am not sure making this configurable makes the most sense. If someone really wants to customize how ddtrace logs are written instead of using this they can just manually configure a log handler for the "ddtrace" logger.

What do you think?

Reply:
I'll mark this as resolved. The configuration table later can be updated to explain how `DD_TRACE_FILE_SIZE_BYTES` can be used to increase the file size.